app/controllers/events_controller.rb

```diff
@@ -0,0 +1,45 @@
+class EventsController < ApplicationController
+  include EventsHelper
+  include UsersHelper
+
+  def index
+    @events = Event.all
+  end
```

Comment:
- [ ] instead of getting the past and upcoming events through the views, use the controller instead ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/past_upcoming_events_requirement.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/associations))